docs/functionality-overview.md

```diff
@@ -0,0 +1,184 @@
+identified as having a Govspeak equivialent, such as a link, are converted to
+their equivalent; elements that are used for non visible data, such as style
+information, are removed; and any other elements are converted to their text
+representation. This conversion is done by using the [turndown] package.
+
+In order to produce a good Govspeak conversion the HTML must be marked up
+[semantically][semantic-html]. This allows the original authors content intent
+to be understood by different application regardless of the formatting. In
+order for this to occur the author must have specified these semantics when
+producing their document and the application used to read the document must
+include this information in copy data to the clipboard.
+
+## Falling back to text input
+
+When pasting this tool will use HTML data if it is available. In most desktop
+browsers there is an option to force this to be treated as plain text - which
+will disable the effects of this tool.
+
+On Windows and Linux machines this can be achieved by using `Ctrl + Shift + v`
+rather than the usual `Ctrl + v` to paste. On macOS the same can be achieved
+with `Cmd + Shift + v`.
+
+On some browsers there are also options on the context menu such as "Paste as
+plain text" on Windows and "Paste and match style" on macOS. These will paste
+in as plain text, bypassing this tool.
+
+## Supported HTML
+
+Only a subset of HTML is converted to Govspeak.
+
+Elements that are converted, but with attributes stripped are:
+
+p, h2, h3, blockquote, pre, code, ol, ul, li, br
+
+Elements that are stripped with allowed attributes are:
+
+a with href, attr with title
+
+Elements that are purposefully stripped for Govspeak consistency, but typically
+allowed in markdown are:
+
+h1, h4, h5, h6, b, strong, em, i, img
+
+Most other elements (most notably tables) are converted to just show text
+within them, except for ones that contain non-content data (such as style,
+script or meta elements) which are removed.
+
+For an example of a best case paste scenario there is an [input example file][],
+containing the supported HTML. This can be pasted into the [tool example][]
+to see the conversion.
+
+## Overview of testing outcomes
+
+In preparing to deploy this functionality we tested a wide range of scenarios,
+what follows is a summary of the findings for expected input sources.
+
+### Common issues
+
+If a document has content separated with line breaks, these may often be
+converted into separate paragraphs which creates an additional line break. This
+happens when the source document stores the separate lines as separate
+paragraphs.
+
+It is expected that users may copy and paste Govspeak/markdown from example
+sources that are represented in HTML (eg [How to publish on GOV.UK][]). Because
+of this no markdown is escaped when it is within HTML. This can impact users
+who wish to use characters that have a syntactic in Govspeak.
```

Comment:
Is there a noun missing after 'syntactic'?

Reply:
D'oh!